flink-libraries/flink-table/src/test/scala/org/apache/flink/table/expressions/ScalarFunctionsTest.scala

```diff
@@ -626,6 +626,33 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "-")
   }
 
+  @Test
```

Comment:
Please add a test case (for demonstration) in `org.apache.flink.table.expressions.SqlExpressionTest`.